tests/microos/live_patching.pm

```diff
@@ -0,0 +1,100 @@
+# SUSE's openQA tests
+#
+# Copyright © 2021 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: smoke test for Live Patching in SLE Micro
+#           1) Get current kernel version
+#           2) Make sure the system has been registered previously
+#           3) Register Live Patching module if not done already
+#           4) Optionally add a kernel update repo
+#           5) Update system and reboot
+#           6) Get new kernel version and compare it
+#
+# Maintainer: qa-c team <qa-c@suse.de>
+
+use base "opensusebasetest";
+use strict;
+use warnings;
+use testapi;
+use transactional;
+use utils qw(zypper_call);
+use version_utils qw(is_sle_micro);
+
+sub register_lp_module {
+    my $arch        = get_required_var('ARCH');
+    my $regcode     = get_required_var('SCC_REGCODE');
+    my $sle_version = get_var('VERSION_SLE', '15.3');
+    my $extensions  = script_output('SUSEConnect --list-extensions');
+    record_info('extensions', $extensions);
+    unless ($extensions =~ m/Live Patching.*Activated/) {
```

Comment:
shouldnt have a softfail for the bug you reported, or something in case the module is not registered. in second thought the module presents in the ay profile. if module is not activated it should be reported immediately. right?

Reply:
The bug was already fixed. Anyway, I add a `die` below if this happens.

Reply:
despite the bug the module should be activated in first place by `autoyast/installation`. die when it fails as a second try. i am not sure about this part. Why?

Reply:
There are 2 use cases:
- DVD flavor with autoyast installation -> everything should be registered and activated during installation
- Image flavor -> there is a scc_registration module already registering the main product. So, we only need to register LP here.

Reply:
then lets move the L40 outside of that block. So it will be checked in both cases. wdyt?

Reply:
Makes sense, let's ensure it for both cases. Thanks!